addon/webextension/selector/ui.js

```diff
@@ -286,7 +286,9 @@ window.ui = (function () { // eslint-disable-line no-unused-vars
     hide: function () {
       window.removeEventListener("scroll", this.onScroll, false);
       window.removeEventListener("resize", this.onResize, true);
-      this.element.style.display = "none";
+      if (this.element) {
```

Comment:
It specifically happens if there's an error during setup, so that the unloader is invoked before the ui is setup.  To test you have to introduce a top-level error into some module.